.gitignore

```diff
@@ -1,4 +1,5 @@
 .idea
-src
 pkg
+src
```

Comment:
you don't really need to ignroe src directory... it shouldn't really be there in the first place. Remove src directory and this as well.

Reply:
Instead of this... we should add `vendor` directory to the ignore list